tutorials/viewports/multiple_resolutions.rst

```diff
@@ -3,21 +3,24 @@
 Multiple resolutions
 ====================
 
-Base resolution
----------------
+Base size
+---------
 
-A base screen resolution for the project can be specified in the project
-settings under ``display``, ``window``
+A base size for the project can be specified in the project settings
+under "Display", "Window".
 
 .. image:: img/screenres.png
 
-However, what it does is not completely obvious. When running on PC, the
-engine will attempt to set this resolution (or use something smaller if
-it fails). On mobile, consoles or devices with a fixed resolution or
-full screen rendering, this resolution will be ignored and the native
-resolution will be used instead. To compensate for this, Godot offers
-many ways to control how the screen will resize and stretch to different
-screen sizes.
+However, what it does is not completely obvious; the engine will *not*
+attempt to switch the monitor to this resolution. Rather, think of this
+setting as the "design size", i.e. the size of the area that you work
+with in the editor. This setting corresponds directly to the size of the
+magenta rectangle in the 2D editor.
```

Comment:
The window size box in 2D view is blue. It's only magenta along the top where it overlaps with the red x-axis.